gapii/cc/spy.cpp

```diff
@@ -372,10 +368,6 @@ std::shared_ptr<StaticContextState> GlesSpy::GetEGLStaticContextState(CallObserv
 }
 
 std::shared_ptr<DynamicContextState> GlesSpy::GetEGLDynamicContextState(CallObserver* observer, EGLDisplay display, EGLSurface draw, EGLContext context) {
-    if (draw == nullptr) {
```

Comment:
Ideally add a comment saying 'draw' may be null, otherwise it will be forgotten in couple of months?

Reply:
Just removed the unused parameters instead.

Reply:
Humm... except they were used. Bugger.